File: 08/ruby/main.rb
puts "How many people? "
people = gets.to_i
puts "How many pizzas? "
pizzas = gets.to_i
pieces = 8 * pizzas

puts "#{people} people with #{pizzas} pizzas."
puts "Each person gets #{pieces / people} pieces of pizza."
puts "There are #{pieces % people} leftover pieces."

File: 08/swift/main.swift
import Foundation

func input() -> String {
	let inputString = NSFileHandle.fileHandleWithStandardInput().availableData
	let strData = NSString(data: inputString, encoding: NSUTF8StringEncoding)!
	return strData.stringByTrimmingCharactersInSet(NSCharacterSet.newlineCharacterSet())
}

print("How many people? ")
let people = Int(input())!
print("How many pizzas do you have? ")
let pizzas = Int(input())!
let pieces = pizzas * 8

print("\(people) people with \(pizzas) pizzas.")
print("Each person gets \(pieces / people) pieces of pizza.")
print("There are \(pieces % people) leftover pieces.")
